data/config-schema/src/main/resources/net/corda/schema/configuration/flow/1.0/corda.flow.json

```diff
@@ -95,7 +97,10 @@
           "maximum": 2147483647,
           "default": 5
         }
-      }
-    }
-  }
+      },
+      "additionalProperties": false
+    },
+    "additionalProperties": false
+  },
+  "additionalProperties": false
```

Comment:
ive a feeling this last one on the root json node _might_ not be doing anything as its not defined within a properties block.